packages/forms-web-app/src/controllers/projects/examination-timetable.js

```diff
@@ -30,14 +30,15 @@ const eventStates = [
 	{ value: 'null', text: '', classes: '' }
 ];
 
+const setTimeToEndOfDay = (date) => new Date(date).setHours(24, 0, 0, 0);
+
 const eventSubmitButtonActive = (timetable) => {
-	const tomorrow = new Date();
-	tomorrow.setDate(tomorrow.getDate() + 1);
+	const currentDate = new Date();
 
 	return (
 		timetable.typeOfEvent === 'Deadline' &&
-		tomorrow < new Date(timetable.dateOfEvent) &&
-		(new Date(timetable.dateTimeDeadlineStart) < getDate() ||
+		currentDate < setTimeToEndOfDay(new Date(timetable.dateOfEvent)) &&
```

Comment:
new date not required as the function does new date

Reply:
Yes done.